ANSWERS.md

```diff
@@ -0,0 +1,9 @@
+### What is the DOM?
+    Docuemnt Object Model basically is every single element in the page placed in the way of a tree.
+    
+### What is an event?
+    Is the action between the page and the web page Ex: a click a drag, dobble click.
+    
+### What is an event listener?
+    An Event Listener is a pice of code that gets fired whenever the event expected is executed.
+    
```

Comment:
You're correct.  The DOM is made up of different nodes that store the page's or document's' data.  It is set out like a tree, with nesting of components within their parent components, etc.